File: 169-majority-element/majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
    Map<Integer,Integer> seen=new HashMap<>();
    int m=0;
    int major=0;
    for(int num:nums){
        seen.put(num,seen.getOrDefault(num,0)+1);
        if(seen.get(num)>m){
            m=seen.get(num);
            major=num;
        }
    }
      
        return major;
    }
}

File: 169-majority-element/majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        seen={}
        m=0
        indx=0
        for i in nums:
            seen[i]=seen.get(i,0)+1
            if m<seen[i]:
                m=seen[i]
                indx=i
        return indx
            

        